src/model.ts

```diff
@@ -64,9 +71,28 @@ export class HubApiModel {
           }
         });
     }
+    return pass;
+  }
 
+  private combineStreamsInSequence(streams: PagingStream[], pass: PassThrough): PassThrough {
+    if (streams.length > 0) {
```

Comment:
Do we ever anticipate this being called with no paging streams at all?

Reply:
@drspacemanphd I don't think we will ever encounter such state. Therefore, I've removed the check.